src/rendering/renderers/shared/system/SharedSystems.ts

```diff
@@ -38,6 +39,7 @@ export const SharedRenderPipes = [
     BlendModePipe,
     BatcherPipe,
     SpritePipe,
+    DomPipe,
```

Comment:
Since this using the DOM we need to treat this like `AccessibilitySystem` and only import it in the browser bundle